README.md

```diff
@@ -34,7 +34,8 @@ A small example fetching the previous page of a hydra paged collection and retur
 ```javascript
 var ldfetch = require('../lib/ldfetch.js'),
     n3 = require('n3');
-var fetch = new ldfetch();
+var options = {headers: [{key: 'Accept-Datetime', value: '2017-03-11T17:00:00.000Z'}]}; // optional
```

Comment:
Why key and value, and why not just `{"headers":{"accept-datatime":"2017-03-11T17:00:00.000Z"}}`?